tests/testapp/tests/test_permission_classes.py

```diff
@@ -229,3 +229,130 @@ def test_view_django_model_permission_with_change_perm(self):
+            response = self.client.put(
+                reverse('test_template_detail', args=[template.id]),
+                data={'name': 'Name changed'},
+                content_type='application/json',
+                **auth,
+            )
+            self.assertEqual(response.status_code, expected_status_codes['update'])
+
+        with self.subTest('Test deleting template'):
+            response = self.client.delete(
+                reverse('test_template_detail', args=[template.id]), **auth
+            )
+            self.assertEqual(response.status_code, expected_status_codes['delete'])
+
+        with self.subTest('Test HEAD and OPTION methods'):
+            response = self.client.head(reverse('test_template_list'), **auth)
+            self.assertEqual(response.status_code, expected_status_codes['head'])
+
+            response = self.client.options(reverse('test_template_list'), **auth)
+            self.assertEqual(response.status_code, expected_status_codes['option'])
+
+    def test_superuser_access_shared_object(self):
+        admin = self._get_admin()
+        token = self._obtain_auth_token(username=admin)
+        self._test_non_superuser_access_shared_object(
+            token,
+            expected_status_codes={
+                'create': 201,
+                'list': 200,
+                'retrieve': 200,
+                'update': 200,
+                'delete': 204,
+                'head': 200,
+                'option': 200,
+            },
+        )
+
+    def test_org_manager_access_shared_object(self):
+        operator = self._get_operator()
+        token = self._obtain_auth_token(username=operator)
+        # First user is automatically owner, so created dummy
+        # user to keep operator as manager only.
+        self._create_org_user(user=self._get_user(), is_admin=True)
+        self._create_org_user(user=operator, is_admin=True)
+        self._test_non_superuser_access_shared_object(
+            token,
+            expected_status_codes={
+                'create': 400,
+                'list': 200,
+                'retrieve': 200,
+                'update': 403,
+                'delete': 403,
+                'head': 200,
+                'option': 200,
+            },
+        )
+
+    def test_org_owner_access_shared_object(self):
+        operator = self._get_operator()
+        token = self._obtain_auth_token(username=operator)
+        self._create_org_user(user=operator, is_admin=True)
+        self._test_non_superuser_access_shared_object(
+            token,
+            expected_status_codes={
+                'create': 400,
+                'list': 200,
+                'retrieve': 200,
+                'update': 403,
+                'delete': 403,
+                'head': 200,
+                'option': 200,
+            },
+        )
+
+    def test_org_user_access_shared_object(self):
+        operator = self._get_operator()
```

Comment:
can we change this test to avoid mentioning operator? I recommend mentioning only "user" and no "operator" which is confusing since operators are usually org managers.